src/main/java/org/folio/circulation/resources/RequestFromRepresentationService.java

```diff
@@ -341,6 +345,21 @@ private Request addUserToLoan(Request request, User user) {
         ATTEMPT_TO_CREATE_TLR_LINKED_TO_AN_ITEM, request));
   }
 
+  private Result<Request> refuseWhenNoRequestDate(Result<Request> request) {
+    return request.next(this::validateIfRequestDateIsPresent)
+      .mapFailure(err -> errorHandler.handleValidationError(err, INVALID_ITEM_ID, request));
+  }
+
+  private Result<Request> validateIfRequestDateIsPresent(Request context) {
+    var requestDate = getDateTimeProperty(context.getRequestRepresentation(), REQUEST_DATE);
+
+    if (requestDate == null) {
+      return failedValidation("Cannot create a request with no requestDate", REQUEST_DATE, null);
```

Comment:
```suggestion
      return failedValidation("Cannot create a request without requestDate", REQUEST_DATE, null);
```

Reply:
I used the same style as it had been already been in the same class. Please see the other error messages above: 
 "Cannot create a request with no instance ID", "Cannot create an item level request with no item ID" etc.

Reply:
got it